orders/models.py

```diff
@@ -0,0 +1,79 @@
+from django.db import models
+from django.utils import timezone
+from daycare.models import DayCare
+from dogowner.models import DogOwner
+
+
+class StatusOptions(models.TextChoices):
+    Pending = 'P', 'Pending'
+    Approved = 'A', 'Approved'
+    Canceled = 'C', 'Canceled'
+    Finished = 'F', 'Finished'
+    OnGoing = 'O', 'On Going'
+
+
+class Order(models.Model):
+    dog_owner_id = models.ForeignKey(DogOwner, on_delete=models.CASCADE)
+    daycare_id = models.ForeignKey(DayCare, on_delete=models.CASCADE)
+    booking_date = models.DateField(auto_now=True)
+    approving_date = models.DateField(blank=True, null=True)
+    starting_date = models.DateField()
+    finishing_date = models.DateField()
+    canceling_date = models.DateField(blank=True, null=True)
+    total_price = models.IntegerField()
+    status = models.CharField(max_length=1,
+                              choices=StatusOptions.choices,
+                              default='P'
+                              )
+
+    def __str__(self):
+        return f"Dog owner ID: {self.dog_owner_id}, daycare ID: {self.daycare_id}, status: {self.status}"
+
+    @staticmethod
```

Comment:
should be @classmethod

Reply:
Doesn't a class method mean that we need to have an Order instance before calling the "create" method?
I've made it static, so the creation of an order is not dependent on a previous order instance.

Reply:
No, a class method is a method of the class, and not of a specific object instance.

Reply:
Changed it.